utility/session_test.go

```diff
@@ -0,0 +1,531 @@
+package utility
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"github.com/pokt-network/pocket/runtime/test_artifacts"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"github.com/pokt-network/pocket/utility/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gonum.org/v1/gonum/stat/combin"
+)
+
+// TECHDEBT: Geozones are not current implemented, used or tested
+
+func TestSession_GetSession_SingleFishermanSingleServicerBaseCase(t *testing.T) {
+	// Test parameters
+	height := int64(1)
+	relayChain := test_artifacts.DefaultChains[0]
+	geoZone := "unused_geo"
+	numFishermen := 1
+	numServicers := 1
+	expectedSessionId := "5acf559f1a3faf3bea7eb692fe51bc1e2e5fb687ede0a6daa7d42399da4aa82b" // needs to be manually updated if business logic changes
```

Comment:
nit: any reason the comment has to be the same line? It makes for quite a long line.